components/signature/overview.md

```diff
@@ -0,0 +1,151 @@
+
+In **Blazor server-side applications**, the Signature uses the **SignalR WebSocket**, which has a default maximum message size of **32 KB**. To work with larger content, increase the max WebSocket message size for the Blazor application:
+
+>caption Program.cs (.NET 6)
+
+<div class="skip-repl"></div>
+
+````CS
+using Microsoft.AspNetCore.SignalR;
+
+var builder = WebApplication.CreateBuilder(args);
+
+//...
+
+// SignalR message size for the Signature
+builder.Services.Configure<HubOptions>(options =>
+{
+    options.MaximumReceiveMessageSize = 1024 * 1024; // 1MB
+});
+````
+
+>caption Startup.cs (.NET 5)
+
+<div class="skip-repl"></div>
+
+````CS
+public void ConfigureServices(IServiceCollection services)
+{
+    //...
+
+    // SignalR message size for the Signature
+    services.Configure<HubOptions>(options =>
+    {
+        options.MaximumReceiveMessageSize = 1024 * 1024; // 1MB
+    });
+}
+````
+
+## Signature Parameters
+
+The following table lists the Signature parameters. Also check the [Signature API Reference](/blazor-ui/api/Telerik.Blazor.Components.Signature).
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+| Parameter | Type and Default&nbsp;Value | Description |
+| --- | --- | --- |
+| `AriaDescribedBy` | `string` | Maps to the `area-describedby` attribute. Use this parameter to establish a relationship between widgets or groups and the text that describes them. |
+| `AriaLabel` | `string` | Maps to the `aria-label` attribute. Use  this parameter if the text that labels the component is not visible. |
+| `AriaLabelledBy` | `string` | Maps to the `area-labelledby` attribute. Use this parameter to reference another element to define its accessible name. |
+| `Class` | `string` | Renders a custom CSS class to the `<div class="k-signature">` element. Use it to [override theme styles]({%slug themes-override%}). |
+| `DebounceDelay` | `int` <br /> `(150)` | Time in milliseconds between the last typed symbol and the internal `oninput` event firing. |
+| `Enabled` | `bool` <br /> `(true)` | Defines if the component is enabled. |
+| `ExportScale` | `double` | The `Width` and `Height` of the component will be multiplied by the value of the `ExportScale` when converting the signature to an image. |
+| `Height` | `string` | Defines the height of the component. Set the value of this parameter in `px`. |
+| `HideLine` | `bool` | Whether the dotted line is rendered. |
+| `Maximizable` | `bool` | Whether the Signature component can be maximized. |
+| `PopupScale` | `double` | The `Width` and `Height` of the component will be multiplied by the value of the `PopupScale` when maximizing (`Maximizable="true"`) the Blazor Signature. |
+| `ReadOnly` | `bool` | Maps to the `readonly` attribute. Use this parameter to put the Signature in read-only state. |
+| `Smooth` | `bool` | Use this parameter to smoothen the signature line. |
+| `StrokeWidth` | `double?` | Set the width of the signature line. |
```

Comment:
What is this signature line?

Reply:
when drawing a signature a line is rendered. Ideally, I would like to call it a stroke, but we can't explain a parameter by its name. What if people are not aware of what a stroke is?

I agree it's not optimal, but I could not come up with a better word.

Reply:
I see. In this case consider "the drawn lines' **thickness** in pixels"